app/views/users/show.html.erb

```diff
@@ -7,3 +8,36 @@
     </ul>
   </article>
 </section>
+
+<div class="d-flex">
+  <section class="user-section mx-3">
+  <h2>friends</h2>
+  <ul class="posts">
+  <%@user.reload.friends.each do |friendship|%>
+  <li><%=friendship.name.capitalize%></li>
+  <%end%>
+  </ul>
+
+  </section>
+  <section class="user-section text-nowrap">
+    <h2>Pending friends</h2>
+    <ul class="posts">
+      <%@user.pending_friends.each do |friend|%>
+        <li>
+          <%=friend.name.capitalize%>
+          <%if current_user != @user%>
+            <%= link_to 'Accept Friend Request',  friendship_path(friend_id:friend.id), class: 'btn btn-primary', method:"patch" %>
+          <%else%>
+            <a href="#" class="btn btn-secondary  disabled" role="button" aria-disabled="true">Pending</a>
+          <%end%>
+        </li>
+      <%end%>
+      <%@user.friend_requests.each do |friend|%>
```

Comment:
There is no need to know the friend requests of a user if the user is not `current_user` 
We can get rid of this too if you make the changes above.